src/lib.rs

```diff
@@ -132,10 +134,13 @@
 //! }
 //! ```
 
-use std::fmt;
-use std::mem::{self, size_of, MaybeUninit};
-use std::ops::{Deref, DerefMut};
-use std::ptr;
+extern crate alloc;
+
+use alloc::boxed::Box;
+use alloc::fmt;
```

Comment:
This should be `core::fmt`, shouldn't it?

Reply:
Yes, it looks like core's fmt has all we need. Changed to `core::fmt`